tailwind.config.ts

```diff
@@ -152,6 +152,9 @@ const config = {
           low: "hsla(var(--background-low))",
           medium: "hsla(var(--background-medium))",
           high: "hsla(var(--background-high))",
+          table: {
+            hover: "var(--table-background-hover)",
+          },
```

Comment:
We can replace this with `background-highlight`.

Reply:
I'll do that shortly